### PR TITLE
Task 01

### PR DESCRIPTION
**Pull request by owsym**

### CODE REVIEW

File: Task/t1.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import os
+from Utils.read_files import read_files
+
+
+file_values = read_files()
+
+Temperatures = {}
```
Comment (Hamma111) [inline Task/t1.py:7]: Variables must always be small case!! (unless they are constants)

Comment (Hamma111) [reply Task/t1.py:7]: do many mistakes. but dont repeat the same mistake again.

File: Task/t1.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import os
+from Utils.read_files import read_files
```
Comment (Hamma111) [inline Task/t1.py:2]: all the folder names must always be small case 
`utils/` 
`tasks/`

File: Utils/__pycache__/read_files.cpython-310.pyc
Comment (Hamma111) [inline Utils/__pycache__/read_files.cpython-310.pyc:1]: Exclude the __pychache__ and *.pyc files in your repo.

Delete the existing pychache folder for it to take effect

File: Utils/read_files.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        with open(os.path.join(dir_path, '..', 'weatherfiles', file_name), "r") as file_reader:
+            file_content = file_reader.read()
+
+            file_values += get_contents_of_one_file(file_content)
+
+    return file_values
+
+file = read_files()
+for f in file:
+    print(f)
```
Comment (Hamma111) [inline Utils/read_files.py:27]: we can probably remove this print(), right?

File: Utils/read_files.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import os
```
Comment (Hamma111) [inline Utils/read_files.py:1]: `readers.py` might be a better name for this file

File: Task/t1.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+
+file_values = read_files()
+
+Temperatures = {}
+for file_value in file_values:
+    if file_value and file_value[1]:
+        Temperatures[file_value[0]] = float(file_value[1])
```
Comment (Hamma111) [inline Task/t1.py:10]: Min temperature and max temperature are present as different columns (and different index) in the file!!

Comment (Hamma111) [reply Task/t1.py:10]: so update this method accordingly

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: Task/t1.py (hunk 1 above)
Comment (owsym) [reply Task/t1.py:7]: ok

File: Task/t1.py (hunk 2 above)
Comment (owsym) [reply Task/t1.py:2]: ok

File: Utils/__pycache__/read_files.cpython-310.pyc
Comment (owsym) [reply Utils/__pycache__/read_files.cpython-310.pyc:1]: ok

File: Utils/read_files.py (hunk 3 above)
Comment (owsym) [reply Utils/read_files.py:27]: yes

File: Task/t1.py (hunk 5 above)
Comment (owsym) [reply Task/t1.py:10]: sorry i dont understand, how you want to change.

**Pull request merged**